### PR TITLE
[Dominion] Fixes - logo, map, footer on profile

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Small fixes on logo link, add footer to profile page, and changed map tiles link..about page

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by karimkawambwa — APPROVED** [review]
looks good to me

**Review by kilemensi — CHANGES_REQUESTED** [review]
Few things @KhadijaMahanga:

 1. There is gap between footer and partners
Screenshot from 2019-03-19 10-58-38
 2. There is a gap between hero and profiles tabs
Screenshot from 2019-03-19 10-58-55

**Review by karimkawambwa — CHANGES_REQUESTED** [review]
As soon as you start typing on the search, it appears broken.

Screen Shot 2019-03-19 at 12 03 12 PM

File: dominion/dominion_ui/public/index.html [hunk 1]
```diff
@@ -74,16 +74,34 @@
...
+
+        var geography = {
+            full_geoid: '',
+            this: {
+                square_kms: '',
+                short_name: '',
+                parents: []
+            }
+        };
     </script>
```
Comment (karimkawambwa) [inline dominion/dominion_ui/public/index.html:105]: Changes to allow yarn start

**Comment by KhadijaMahanga** [comment]
@kilemensi pushed fixes

**Comment by KhadijaMahanga** [comment]
> As soon as you start typing on the search, it appears broken.
> 
> Screen Shot 2019-03-19 at 12 03 12 PM

@karimkawambwa is this through `yarn start`? I can't reproduce it

**Review by kilemensi — APPROVED** [review]

**Pull request merged**